app-modules/assistant/src/Filament/Pages/PersonalAssistant.php

```diff
@@ -91,6 +86,30 @@ public function mount(): void
         );
     }
 
+    #[Computed]
```

Comment:
What is the purpose of refactoring to Livewire computed properties? Does this provide us with some benefit that the original property implementation did not?

Unfortunately, this seems to be one of the biggest causes of the linting errors.

Reply:
I was having problems with losing the sorting anytime the collection changed: https://livewire.laravel.com/docs/properties#eloquent-constraints-arent-preserved-between-requests

I was able to fix the linting with a class property:
```php
/**
 * @property EloquentCollection $chats
 */
```